Purchases/Public/Purchases.swift

```diff
@@ -221,22 +221,14 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
 
     fileprivate static let initLock = NSLock()
 
-    convenience init(apiKey: String, appUserID: String?) {
```

Comment:
This was `internal` and it wasn't used. It was also redundant with the next constructor with default parameters.

Reply:
nice catch, that's a leftover from pre-swift migration days